CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Change Log
 
+## [Unreleased]
+
+#### Warning - Potentially breaking change
+- Restivus used to store the account login token in the user document: services.resume.loginTokens.token
+- Restivus now stores the account login token as a hashed token, in the user document: services.resume.loginTokens.hashedToken
+- This matches Meteor Accounts package
```

Comment:
Please highlight any code you use with markdown. I left a similar comment below. Helps me sleep better at night. So services.resume.loginTokens.token should be `services.resume.loginTokens.token`